thirdparty/embree/common/simd/arm/sse2neon.h

```diff
@@ -2080,8 +2082,16 @@ FORCE_INLINE void *_mm_malloc(size_t size, size_t align)
         return malloc(size);
     if (align == 2 || (sizeof(void *) == 8 && align == 4))
         align = sizeof(void *);
-    if (!posix_memalign(&ptr, align, size))
+// -- GODOT start --
+#if defined(_WIN32)
+    ptr = _aligned_malloc(size, align);
+	if (ptr)
         return ptr;
+#else
+	if (!posix_memalign(&ptr, align, size))
+        return ptr;
+#endif
+// -- GODOT end --
```

Comment:
There's mixed tabs and spaces indentation.

Reply:
Fixed.